docker/docker-compose-multiclusters.yml

```diff
@@ -0,0 +1,62 @@
+version: '3'
+services:
+  cassandra:
+    image: cassandra:3.11
+    ports:
+      - "9042:9042"
+  statsd:
+    image: graphiteapp/graphite-statsd
+    ports:
+      - "8080:80"
+      - "2003:2003"
+      - "8125:8125"
+      - "8126:8126"
+  cadence:
+    image: ubercadence/server:master-auto-setup
+    ports:
+     - "7933:7933"
+     - "7934:7934"
+     - "7935:7935"
+     - "7939:7939"
+    environment:
+      - "CASSANDRA_SEEDS=cassandra"
+      - "STATSD_ENDPOINT=statsd:8125"
+      - "DYNAMIC_CONFIG_FILE_PATH=config/dynamicconfig/development.yaml"
```

Comment:
The replication has to be config from the yaml file. Take a look at the development_active.yaml and development_standby.yaml

Reply:
Yes, it uses config_template.yaml to generate config file. This environment variable is set to get dynamic config.

Reply:
Oh, I see. This is a dynamic config yaml. Then it looks good to me.